apps/schematic/api/project.json

```diff
@@ -15,10 +15,13 @@
       "executor": "nx:run-commands",
```

Comment:
hmm what is project.json used for?

Reply:
The projects in this monorepo are managed with a tool called `nx`. If a folder in this monorepo has a file `project.json`, then Nx will detect this folder as a "project". The file `project.json` defines the tasks that are implemented for a given project. These tasks are then executed with Nx with the command `nx <task> <project>` or `nx run <project>:<task>`. E.g., `nx prepare-python schematic-api` (or `nx run schematic-api:prepare-python`), `nx build schematic-api-description` (or `nx run schematic-api-description:build`), etc.

Reply:
Yes. This makes sense. Thanks Thomas. I was able to run schematic-api-description too